packages/preset-polaris/src/index.js

```diff
@@ -0,0 +1,43 @@
+import base from '@theme-ui/preset-base'
+
+const text = '"San Francisco", Roboto, "Segoe UI", "Helvetica Neue", sans-serif'
```

Comment:
"San Francisco" isn't the correct font name. Instead:
```suggestion
const text = 'system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", sans-serif'
```